fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/CheckAfterRewrite.java

```diff
@@ -45,11 +48,31 @@ public class CheckAfterRewrite extends OneAnalysisRuleFactory {
     public Rule build() {
         return any().then(plan -> {
             checkAllSlotReferenceFromChildren(plan);
+            checkNoUnbounded(plan);
```

Comment:
we have check bound in org.apache.doris.nereids.rules.analysis.CheckAnalysis#checkBound.
But it only check unbound slot.
If u want to check UnboundFunction, update it~

Reply:
done